src/UI/Component/Popover/Popover.php

```diff
@@ -98,4 +98,18 @@ public function getShowSignal();
 	 * @return ReplaceContentSignal
 	 */
 	public function getReplaceContentSignal();
+
+
+	/**
+	 * Get a popover which can be used in fixed places such as the main menu
+	 *
+	 * @return $this
+	 */
+	public function withFixedPosition();
```

Comment:
I see that this is necessary at the current stage of the UI-Framework, since the nor the triggerer nor the popover will now wheter they are fixed or not. However I strongly hope that this function will at some point only be used in other larger components (such as some Tob bar component) which now, that they are fixed and therefore only generate fixed popovers. Nevertheless they will need this also.